apps/extension/webpack.prod-config.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * This config overrides the env file for the extension and changes the output directory
+ */
+
+import dotenv from 'dotenv';
+import path from 'node:path';
+import config from './webpack.config.js';
+
+dotenv.config({ path: '.env', override: true });
```

Comment:
question: would it make it simpler if we didn't have an env file at all and just put the ids in the respective prod vs beta webpack configs?